biopolymers.csl

```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded" default-locale="en-US">
+  <info>
+    <title>Biopolymers</title>
+    <title-short>Biopolymers</title-short>
+    <id>http://www.zotero.org/styles/Biopolymers</id>
```

Comment:
this is case sensitive -- needs to be `http://www.zotero.org/styles/biopolymers`